app/src/main/java/com/mapbox/mapboxsdk/plugins/testapp/activity/scalebar/ScalebarActivity.kt

```diff
@@ -0,0 +1,67 @@
+package com.mapbox.mapboxsdk.plugins.testapp.activity.scalebar
+
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import com.mapbox.mapboxsdk.camera.CameraUpdateFactory
+import com.mapbox.mapboxsdk.maps.Style
+import com.mapbox.mapboxsdk.plugins.testapp.R
+import com.mapbox.pluginscalebar.ScaleBar
+import kotlinx.android.synthetic.main.activity_scalebar.*
+
+/**
+ * Activity showing a scalebar used on mapview.
+ */
+class ScalebarActivity : AppCompatActivity() {
+    private var scaleBar: ScaleBar? = null
```

Comment:
when using `import kotlinx.android.synthetic.main.activity_scalebar.*` you can reference your scaleBar directly in your code, no need to add a field for it.

Reply:
this also removes the requirement of needed to add `scaleBar!!` all over the code :)